lib/actions/action-maintain-contact.ts

```diff
@@ -32,17 +36,17 @@ const handler: ActionFile['handler'] = async (
 		'contact@1.0.0',
 	)) as TypeContract;
 	assert.INTERNAL(
-		request.context,
+		request.logContext,
 		typeCard,
-		context.errors.WorkerNoElement as any, // TS-TODO fix this
+		workerErrors.WorkerNoElement,
 		'No such type: contact',
 	);
 
 	const originCard =
 		card.data.origin &&
-		(isUUID(card.data.origin)
-			? await context.getCardById(session, card.data.origin)
-			: await context.getCardBySlug(session, card.data.origin));
+		(isUUID(card.data.origin as string)
```

Comment:
This irks me a lot even though I understand why 😔

Reply:
Maybe it would be better to do some assertions above this line and pull `card.data.origin` out into its own string variable and then use that instead.

Reply:
sounds better